public/routing/index.plugin.ts

```diff
@@ -22,3 +22,41 @@ const routingConfig: RoutingConfiguration = {
 };
 
 export default routingConfig;
+
+export const createPluginLinkBuilder: LinkBuilderFactory = (appKey?: string) => {
```

Comment:
A comment how this needs to take all the URL params and add `ac[APP_KEY][param]` and include the project id and project key.

__BTW__ why are project id and project key provided?  Is this so plugins could use them as the context for the project folks are coming from?  For example, maybe we want to show a report automatically from the project key?

Reply:
Jira adds these when we load the application in jira. I'm preserving them. I've added a comment articulating that as well

Reply:
we __might__ be able to muck about with them to change the project scope, but since jira adds I don't think we should